### DISCUSSION TITLE
Add built in config preset for @pulumiverse

### DISCUSSION THREAD

**Discussion by 1oglop1** [renovatebot/renovate#31864] [Suggest an Idea]
### Tell us more.

Hi, the group `pulumi` already exists https://docs.renovatebot.com/presets-group/#grouppulumi
however pulumi also has community packages from `@pulumiverse`.
It would be great to add pulumiverse group too.

https://github.com/pulumiverse

**Comment by RahulGautamSingh** [comment]
You can contribute one here: https://github.com/renovatebot/renovate/edit/main/lib/config/presets/internal/group.ts

**Reply by 1oglop1** [reply]
Could you please elaborate on what I need to do in this case? 
I mean process-wise, I know where to edit the code. 

OT:
I already have one discussion open #31778 and it appears to be stuck in the process, even though I think I provided everything required. Another PR has been waiting for review https://github.com/renovatebot/github-action/pull/866#pullrequestreview-2357876779

**Reply by RahulGautamSingh** [reply]
This will help https://github.com/renovatebot/renovate/blob/main/lib/data/readme.md#group-related-packages-monorepojson

If it doesn't, then I will comment in more detail and also update this doc to be more helpful
